### DISCUSSION TITLE
How do I Autoplay the next video in my Playlist?

### DISCUSSION THREAD

**Discussion by Lasakro** [FreeTubeApp/FreeTube#2364] [Q&A]
When I play a video in my Playlist, Freetube will the play next the video in the Up Next (suggested list). With the Up Next Autoplay off the player stops at the end of the video I first selected from my playlist.

I'm using v16, 
Autoplay Videos = off
Autoplay Playlists = on
Play Next Video = on

Is the function of autoplaying a playlist not a feature of Freetube?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@Lasakro just to clarify with **your playlist** do you mean the videos listed in this tab?
![playlist](https://user-images.githubusercontent.com/73130443/176737440-bcd01e88-782c-421d-9afc-3e16e7113b98.PNG)

**Reply by Lasakro** [reply]
That is correct. When in the Playlist window the attached dialog box can be found. The wording is a little vague. It states "When the work has finished, all videos currently here will be migrated to a 'Favorites' playlist.

I initially thought this was "My work" by adding videos to the window. Now I'm reading this as "Your work" meaning that the program doesn't have this feature at this point. Is the later correct?
![Freetube Playlists](https://user-images.githubusercontent.com/14164409/176740263-7d36ed24-3eea-44f2-bdb0-22ed858c1b85.png)
?

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
You are correct. Our playlist tab only contains videos that you saved but as u can see u cant group videos in an actual list yet. This has yet to be implemented.

**Reply by Lasakro** [reply]
I appreciate the fast response. I'll see if I can "Close" this topic.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Nothing has to be done to close this topic. I have marked this as Answered so we consider it closed :)
